### ISSUE TITLE
Task: Write policy surrounding inactive chapter

### ISSUE THREAD

**Issue by Mariatta** [pyladies/project-admin-logisitics#10]
# TL;DR

The policy should answer the following questions:
1. Why is this policy necessary?
2. What are criteria when we would consider a chapter inactive?
3. How do we want to enforce it?
4. For accountability, how do we (administrators) document the deactivation?

# Details

## 1. Why is this policy necessary?

- We need to have accurate information of our chapter directory, where are pyladies, where are our chapters.
- There are indeed 100s of chapters who has not logged in to their chapter email address
- Each PyLadies chapter uses resources in many forms: they might have Pro meetup account from The PSF, they may have website hosted, they might require funding from The PSF.
- PyLadies admins received numerous inquiries from community members regarding local chapters that turned out to be inactive, taking up additional resource/volunteer time from Global leadership and administrator team to address these inquiries

## 2. What are criteria when we would consider a chapter inactive?

Some determining factors:

- has not logged in to their chapter email after 2 years
- no meetup after 2 years
- organizers non responsive after 2 months.

## 3. How do we want to enforce it?

- we write to chapters once a year to request status update
- from GSuite admin we can review email login activity once a year. 
- a community member can notify us if they noticed a chapter has become inactive. (e.g. they 
want to find a meetup, but was unable to reach anyone). 

  In any of the above situations, we will then contact organizers of the inactive chapter, through 
  chapter email, and to their personal email address we have on file. 

## 4. For accountability, how do we (administrators) document the deactivation?

We can record all activities, including attempt to contact organizer (and the result) as an .md file 
in this repo.

# Things to document:

1. how do we find out about inactive chapter? was it through GSuite, or through an email from community member?

2. what were our attempts to contact the organizer, before deactivating?
Which email address we tried? How many times we tried to reach them?

3. any response from organizers?

4. date we deactivate the chapter.

# Proposed Email template

To: {chapter}@pyladies.com, {personal organizer email address}, ...
cc: info@pyladies.com

Subject: Is  {chapter} PyLadies inactive?

Dear {Organizer name} and {chapter} PyLadies , 

This is {admin} from PyLadies. You are listed as the original organizers of {chapter} PyLadies.
Thanks your work with PyLadies!

We noticed there has been no login to {chapter} pyladies email account since {last login date}.

**Per our current policy, such chapter is considered inactive.**

Please reply to the following.

1. **What's the status of {chapter} PyLadies?** Is the chapter still active?

2. **Would you like to continue running the chapter?** Or would you like to hand it over to new organizers?

3. Is there anything we can help from global PyLadies leadership team to better support {chapter} community?

We understand personal situations can change. If you're no longer able to lead the {chapter} PyLadies, that is totally fine. We thank you for your work so far.

Looking forward to hearing back from you.

**If we don't hear back after 2 months**, we will consider the chapter inactive, and will deactivate the account or hand it over to new organizers. However, I would prefer to hear back from you first. Please reply as soon as you have a chance, and let us know how we can further help.

Thank you.

{admin} on behalf of PyLadies

**Comment by lorenanicole** [comment]
# Criteria for marking chapter inactive

- Chapter has not registered for the Chapter Directory or have failed to confirm details; we will submit confirmation to chapters yearly to confirm details of what they registered in Chapter Directory
- Chapter has not posted an activity on their event platforms (e.g. twitter, meetup, eventbrite) in 1+ year ; will use details from Chapter Directory to confirm the activity platforms else PyLadies Global will use the scripts provided in the Global Organizing repo to as best as possible infer activity. 

**When criteria are not met**

- If 1.) chapter directory information is missing and 2.) PyLadies Global is unable to confirm activity within the last 12 calendar months, then PyLadies Global will email the chapter telling them they have 30 calendar days to re-claim the chapter. 
- If no response, the email account credentials will be reset and an automatic email response will be set reading, `This chapter is inactive, please contact info@pyladies.com if you are looking to re-claim this chapter or want to coordinate with PyLadies for PyLadies work. Thank you.`. 
- If a response is obtained, some official "warning" will be issued indicating what will happen if the chapter lapses again.

# Making a Chapter Inactive

Like the above states here are the steps to take:

- If no response, the email credentials are reset and passed back to the admins of this group or for PyLadies Global
- Set a email response like `This chapter is inactive, please contact info@pyladies.com if you are looking to re-claim this chapter or want to coordinate with PyLadies for PyLadies work. Thank you.`
- Update PyLadies Chapter Directory, if needed
- Update PyLadies Inactive Chapter Form including explanation of what happened, dates of communication, and who worked on it
- Keep the chapter open for 3 calendar months, if no one reclaims it delete the account.

**Comment by lorenanicole** [comment]
@Mariatta re: "Proposed Template" I would suggest we do `30 days` rather than 2 months. Ideally this add some urgency in will prompt people to reply. Additionally as the frequency of PyLadies Global is on a monthly basis, I think it is fair to match the frequency of their meeting schedule since the hope is to have project teams reporting monthly to PyLadies Global (so likely good to have a consistent schedule for all the things).

**Comment by amadikwajoyn** [comment]
@Mariatta that's a nice write-up and proposal you have there, but I concur with @lorenanicole , I think 2 months is too far for email confirmation and 2years is too big to dictate inactive chapters... I suggest a month email confirmation and 12 months inactive chapters dictation...

**Comment by lorenanicole** [comment]
I wrote up a [script here](https://github.com/pyladies/global-organizing/tree/main/scripts/update-active-chapters) that merges data from:

- Google pyladies accounts on all registered emails
- All pyladies chapter data in the website `www/config.yml` and then grabbed any meetup data from what was specified here
- All pyladies data in the chapter directory we began building Nov 2019

Of the 221 accounts registered in the PyLadies Google account we have 92 then that are deemed inactive because:

- They haven't used their email in the last 1 year
- Never logged into their email
- Are not registered in the chapter directory

Proposal:

- Email one last time asking them to confirm intent to stay active, if they wish to they need to re-register in the Chapter directory. This has to be done within 30 days.
- For now, we remove these chapters from the PyLadies website until this is resolved.
- If after 30 days from email no response happened, we reset these passwords and hold the accounts.
